populus/config/versions.py

```diff
@@ -5,6 +5,7 @@
 V5 = '5'
 V6 = '6'
 V7 = '7'
+V7_VIPER = '7-viper'
```

Comment:
This appears to be un-used?

Reply:
Ha yes! Must've slipped in from when I was still figuring out things ;)